src/app_service/service/chat/service.nim

```diff
@@ -340,3 +340,91 @@ QtObject:
       let errDesription = e.msg
       error "error: ", errDesription
       return
+
+  method addGroupMembers*(self: Service, chatId: string, pubKeys: seq[string]) =
+    try: 
+      let response = status_chat.addGroupMembers(chatId, pubKeys)
```

Comment:
You can discard the response instead. Same for most of them below

Reply:
Still relevant, the `response` is not used here or below.

Reply:
@borismelnik I agree with @jrainville here.